code/game/objects/items.dm

```diff
@@ -602,7 +612,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 			var/index = blood_splatter_index()
 			var/icon/blood_splatter_icon = blood_splatter_icons[index]
 			if(blood_splatter_icon)
-				cut_overlay(blood_splatter_icon)
+				cut_overlay(blood_splatter_icon)
```

Comment:
Hey Hold up I don't think I actually "changed" that beyond moving a line down.

So that was there BEFORE I made my changes, so excuse me if I'm loathe to fiddle with it.